tests/framework/utils/snapshot.go

```diff
@@ -112,20 +112,33 @@ func (k8sh *K8sHelper) DeleteSnapshotController() error {
 
 // snapshotCRD can be used for creating or deleting the snapshot CRD's
 func (k8sh *K8sHelper) snapshotCRD(action string) error {
+	// setting validate=false to skip CRD validation during create operation to
+	// support lower Kubernetes versions.
+	args := func(crdpath string) []string {
+		a := []string{
+			action,
+			"-f",
+			crdpath,
+		}
+		if action == "create" {
+			a = append(a, "--validate=false")
```

Comment:
Could this be added only for failing versions (1.18)?

Reply:
it's not a problem because these are the templates not maintained by us or installed by us, this is getting installed only in the test environment.

Reply:
Even if it's only installed on test envs, the test serves as a validation that the deployment works. So why not make it an exception for the impacted k8s release? Would it be useful to discover issues? Or do we just don't care?

Reply:
@leseb Even if we discover the issues we cannot fix them. these yaml are pulled from the released version (which is already tested and released) as we just skip the kubectl validation, if anything wrong with the tests it will fail we can easily identify the issues. IMO we don't need to care about the kubernetes version here.